src/Events/Integrations/Plugins/Tickets_Wallet_Plus/Controller.php

```diff
@@ -0,0 +1,130 @@
+<?php
+
+namespace TEC\Events\Integrations\Plugins\Tickets_Wallet_Plus;
+
+use TEC\Common\Integrations\Traits\Plugin_Integration;
+use TEC\Events\Integrations\Integration_Abstract;
+use Tribe__Template;
+
+/**
+ * Class Controller
+ *
+ * @since   6.1.1
+ *
+ * @package TEC\Events\Integrations\Plugins\Tickets_Wallet_Plus
+ */
+class Controller extends Integration_Abstract {
+	use Plugin_Integration;
+
+	/**
+	 * @inheritDoc
+	 */
+	public static function get_slug(): string {
+		return 'event-tickets-wallet-plus';
+	}
+
+	/**
+	 * @inheritDoc
+	 */
+	public function load_conditionals(): bool {
+
+		return true;
+
+		// @todo @codingmusician: Fix the logic below and uncomment.
+		// if ( ! class_exists( '\TEC\Tickets_Wallet_Plus\Controller', false ) ) {
```

Comment:
Do we want to address this now? If not, do we need a ticket?

Reply:
We have a ticket for it. ETWP-50